tests/functional/kata-deploy/kata-deploy.bats

```diff
@@ -135,52 +94,8 @@ setup() {
 teardown() {
 	pushd "${repo_root_dir}"
 
-	if [ "${KUBERNETES}" = "k0s" ]; then
-		deploy_spec="-k \"tools/packaging/kata-deploy/kata-deploy/overlays/k0s\""
-		cleanup_spec="-k \"tools/packaging/kata-deploy/kata-cleanup/overlays/k0s\""
-	elif [ "${KUBERNETES}" = "k3s" ]; then
-		deploy_spec="-k \"tools/packaging/kata-deploy/kata-deploy/overlays/k3s\""
-		cleanup_spec="-k \"tools/packaging/kata-deploy/kata-cleanup/overlays/k3s\""
-	elif [ "${KUBERNETES}" = "rke2" ]; then
-		deploy_spec="-k \"tools/packaging/kata-deploy/kata-deploy/overlays/rke2\""
-		cleanup_spec="-k \"tools/packaging/kata-deploy/kata-cleanup/overlays/rke2\""
-	elif [ "${KUBERNETES}" = "microk8s" ]; then
-		deploy_spec="-k \"tools/packaging/kata-deploy/kata-deploy/overlays/microk8s\""
-		cleanup_spec="-k \"tools/packaging/kata-deploy/kata-cleanup/overlays/microk8s\""
-	else
-		deploy_spec="-f \"tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml\""
-		cleanup_spec="-f \"tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml\""
-	fi
-
-	kubectl delete ${deploy_spec}
+	helm uninstall --namespace=kube-system kata-deploy
 	kubectl -n kube-system wait --timeout=10m --for=delete -l name=kata-deploy pod
```

Comment:
Would `--wait` help here?

Reply:
Not really, as `--wait` does not take effect on single replicas and maxUnavailable=1 DaemonSets like kata-deploy on CI. So we wait for pods being Running in the "tradicional" way.